pkgs/applications/office/wpsoffice/default.nix

```diff
@@ -100,6 +145,11 @@ stdenv.mkDerivation rec {
     "libQtCore.so.4"
     "libQtNetwork.so.4"
     "libQtXml.so.4"
+    # WPS 365 specific: file manager integration. Not needed
+    "libnautilus-extension.so.1"
+    "libcaja-extension.so.1"
+    # WPS 365 specific: ukylin specific lib. Not needed on NixOS
```

Comment:
```suggestion
```
peony is also a file manager like nautilus

Reply:
> peony is also a file manager like nautilus

I see. However, from my experience, file manager related libraries are only used to inject some entries (like cloud drive, etc.) into file manager, which should not breaks major usability. Maybe we could leave this as-is until it is indeed important.